suite-common/firmware/src/firmwareReducer.ts

```diff
@@ -86,6 +87,17 @@ export const prepareFirmwareReducer = createReducerWithExtraDeps(initialState, (
             state.cachedDevice = payload;
         })
         .addCase(deviceActions.addButtonRequest, extra.reducers.addButtonRequestFirmware)
+        .addCase(deviceActions.connectDevice, (state, { payload: { device } }) => {
```

Comment:
Similar condition in [deviceReducer](https://github.com/trezor/trezor-suite/blob/b870aa9ff159888c2d813cd5d1aa318b7f7d9f4d/suite-common/wallet-core/src/device/deviceReducer.ts#L648), but this is simpler because we don't care about unacquired devices. Then it's just one action we need to match, so we can use the simpler `addCase` instead of `addMatcher`